### PR TITLE
CHI-2358: Add extra line for describing when the status was last updated

### PR DESCRIPTION
**Pull request by stephenhand**
## Description

* Add  to the EditSummary modal.
* Use dedicated updateCaseStatus endpoint.

### Checklist
- [ ] Corresponding issue has been opened
- [ ] New tests added
- [ ] Feature flags added
- [ ] Strings are localized
- [ ] Tested for chat contacts
- [ ] Tested for call contacts

### Related Issues
Fixes #....

### Verification steps

See ticket - requires latest HRM v1.14 & enable_last_case_status_update_info set to true

### CODE REVIEW

File: plugin-hrm-form/src/states/case/saveCase.ts [hunk 1]
```diff
@@ -39,7 +39,18 @@ export const createCaseAsyncAction = createAsyncAction(
...
+    let updatedCase;
+    if (Object.keys(otherEdits).length) {
+      updatedCase = await updateCase(caseId, otherEdits);
+    }
+    if (status) {
+      updatedCase = await updateCaseStatus(caseId, status);
+    }
+    if (!updatedCase) {
+      updatedCase = await getCase(caseId); // If the update is empty, just refresh the case from the backend.
+    }
```
Comment (GPaoloni) [inline plugin-hrm-form/src/states/case/saveCase.ts:52]: Why would this be true? If none of `updateCase` nor `updateCaseStatus` has been called why is the action called in first place?

File: plugin-hrm-form/src/types/types.ts [hunk 2]
```diff
@@ -277,7 +281,8 @@ export type FeatureFlags = {
   enable_client_profiles: boolean; // Enables Client Profiles
   enable_case_merging: boolean; // Enables adding contacts to existing cases
   enable_confirm_on_browser_close: boolean; // Enables confirmation dialog on browser close when there are unsaved changes
-  enable_separate_timeline_view: boolean; // Enables confirmation dialog on browser close when there are unsaved changes
+  enable_separate_timeline_view: boolean; // Enables a limited inline case timelinbe with a link to the full timeline
+  enable_last_case_status_update_info: boolean; // Enables showing the time, user and changed status of the most recent case status update on the 'Edit Case Summary' page
```
Comment (GPaoloni) [inline plugin-hrm-form/src/types/types.ts:285]: Aren't this better suited as config flags?

**Review by GPaoloni — APPROVED** [review]
LGTM :rocket:

File: plugin-hrm-form/src/types/types.ts (hunk 2 above)
Comment (stephenhand) [reply plugin-hrm-form/src/types/types.ts:285]: I don't think so - I don't see anyone wanting to keep these disabled long term

File: plugin-hrm-form/src/states/case/saveCase.ts (hunk 1 above)
Comment (stephenhand) [reply plugin-hrm-form/src/states/case/saveCase.ts:52]: It probably shouldn't be, but it could, so I think we should handle the possibility?

**Pull request merged**